primitives/linux-port/column.cpp

```diff
@@ -1465,14 +1537,20 @@ void vectorizedFiltering(NewColRequestHeader* in, ColResultHeader* out, const T*
     // outside the scope of the memory allocated to out msg.
     // vectWriteColValues is empty if outputMode == OT_RID.
     uint16_t valuesWritten = vectWriteColValues<T, VT, OUTPUT_TYPE, KIND, HAS_INPUT_RIDS>(
-        simdProcessor, writeMask, nonNullOrEmptyMask, validMinMax, ridOffset, dataVecTPtr, dstArray, Min, Max,
+        simdProcessor, writeMask, nonNullOrEmptyMask, validMinMax, ridOffset, dataVecTPtr, dstArray, min, max,
         in, out, ridDstArray, ridArray);
     // Some outputType modes saves RIDs also. vectWriteRIDValues is empty for
     // OT_DATAVALUE, OT_BOTH(vectWriteColValues takes care about RIDs).
     valuesWritten = vectWriteRIDValues<T, VT, OUTPUT_TYPE, KIND, HAS_INPUT_RIDS>(
-        simdProcessor, valuesWritten, validMinMax, ridOffset, dataVecTPtr, ridDstArray, writeMask, Min, Max,
+        simdProcessor, valuesWritten, validMinMax, ridOffset, dataVecTPtr, ridDstArray, writeMask, min, max,
         in, out, nonNullOrEmptyMask, ridArray);
 
+    if constexpr (KIND != KIND_TEXT)
```

Comment:
I will not force you to rewrite removing this if condition but as you can see you have two functions + additional if constexpr. If you just use SFINAE or concepts based template you would just have 2 specializations.

Reply:
For me it feels like vectorizedTextUpdateMinMax is a bit different from vectorizedUpdateMinMax in how it does things, so I liked the idea of having functions with different logic (and different number of arguments) have different names.